CardScan/Classes/TestingImageExtension.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  TestingImageExtension.swift
+//  CardScan
+//
+//  Created by Jaime Park on 12/3/19.
+//
+
+import Foundation
+
+extension ScanBaseViewController {    
+    func testingImagesSizeCheck() -> [CGImage?]{
```

Comment:
Is there a cleaner way to handle this. I think what you have looks correct, but it's a little hard to follow / more verbose that it might have to be. Play around with it and see if you can come up with anything that you think looks cleaner